restli-tools/src/main/java/com/linkedin/restli/tools/annotation/schemaAnnotationHandlerUtil.java

```diff
@@ -0,0 +1,78 @@
+/*
+   Copyright (c) 2020 LinkedIn Corp.
+
+   Licensed under the Apache License, Version 2.0 (the "License");
+   you may not use this file except in compliance with the License.
+   You may obtain a copy of the License at
+
+       http://www.apache.org/licenses/LICENSE-2.0
+
+   Unless required by applicable law or agreed to in writing, software
+   distributed under the License is distributed on an "AS IS" BASIS,
+   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+   See the License for the specific language governing permissions and
+   limitations under the License.
+*/
+package com.linkedin.restli.tools.annotation;
+
+import com.linkedin.data.schema.annotation.SchemaAnnotationHandler;
+import java.net.URL;
+import java.net.URLClassLoader;
+import java.nio.file.Paths;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+import java.util.StringTokenizer;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import static com.linkedin.data.schema.resolver.FileDataSchemaResolver.*;
+
+
+public class schemaAnnotationHandlerUtil
```

Comment:
Added JavaDoc, capitalized class name.
Keep this class name as "SchemaAnnotationHandlerUtil"
Updated the other one to "SchemaAnnotationHandlerClassUtil", since that one is used to get schema annotation handler class names.